spec/unit/helpers/check_helper_spec.rb

```diff
@@ -48,4 +49,25 @@
     params.setup_filters(PARAMS_3)
     expect { params[:test_param_three] }.to raise_error(/not defined in the specification/)
   end
+
+  context 'when key contains upper-case letters' do
+    let(:spec) do
+      GoodData::LCM2::BaseAction.define_params(self) do
+        description 'Testing param'
+        param :upper_case_param, instance_of(GoodData::LCM2::Type::StringType)
+      end
+    end
+
+    before { params.setup_filters(spec) }
+
+    it 'fetching works with lower case' do
```

Comment:
Similar code found in check_helper_spec.rb:63, check_helper_spec.rb:68 (mass = 36)